tests/integ/scala/test_file_operation_suite.py

```diff
@@ -24,20 +25,25 @@ def random_alphanumeric_name():
     )
 
 
+def onerror(func, path, exc_info):
+    sleep(1 + random.random() * 4)
```

Comment:
does it retry automatically after onerror is called?

Reply:
`onerror` is called after the first error is encountered. So in a way we are giving two tries for `rmtree` to work

Reply:
so if the second try still fails, it will fail?

Reply:
yes

Reply:
I prefer removing the random seconds. If it fails again we have no clue how long time is good.
If we want to wait more time, retry every 1 second and retry 3 or 5 times is better.

Reply:
how many times should we retry?

Reply:
I misread your comment. You meant try every 1 sec with 3-5 retries.